portfolio/src/main/webapp/index.html

```diff
@@ -14,27 +14,38 @@ <h1 id="intro">Hi. I'm Zian.</h1>
                     <div id="external-links">
                         <a href="mailto:zianchen774@gmail.com" target="_blank">
                             <img class="icon" src="./images/mail_icon.png" 
-                                alt="mail_icon"/>
+                                 alt="Mail icon"/>
                         </a>
                         <a href="http://github.com/zianc">
                             <img class="icon" src="./images/github_icon.png" 
-                                alt="github_icon"/>
+                                 alt="Github icon"/>
                         </a>
                         <a href="http://linkedin.com/in/zianchen">
                             <img class="icon" src="./images/linkedin_icon.png" 
-                                alt="linkedin_icon"/>
+                                 alt="LinkedIn icon"/>
                         </a>
                     </div>
                 </div>
                 <div id="path">
                     <div class="path-stop">
                         <img id="path-stop-img-me" src="./images/profile_picture.jpg" 
                             alt="profile_picture"/>
-                        <p class="path-stop-desc">Thanks for dropping by! My name is Zian Chen, a soon-to-be third year student at UCLA, working to get a Bachelor's in Computer Science. My path to where I am now has not been easy, but along the way I've picked up multiple different skills and, arguably more importantly, how to deal with failure, learn the value of criticism, and rely not only on myself but those around me. Keep scrolling to track a few stops along the way! </p>
+                        <p class="path-stop-desc">
+                            Thanks for dropping by! My name is 
```

Comment:
Is there a reason why the first line of the paragraph is shorter than the rest?

Reply:
The first line used to be inline with the <p> tag, and I moved it to a new line which reduced its length.

Reply:
The common practice is to reflow the text. Please do that. :-)

Reply:
I just realized that I fixed it in the second pull request already! Will fix it here as well.

Reply:
Fixed!